dev/io.openliberty.microprofile.health.3.1.internal/resources/io/openliberty/microprofile/health31/resources/Health31.nlsprops

```diff
@@ -0,0 +1,55 @@
+
+
+#-----------------------------------------------------------------------------------------------------------------------------
+# Emergency Health error message
+#-----------------------------------------------------------------------------------------------------------------------------
+OSGI_SERVICE_ERROR=CWMMH0000E: The {0} OSGi service is not available.
+OSGI_SERVICE_ERROR.explanation=This internal server error is thrown because the specified service is not available.
+OSGI_SERVICE_ERROR.useraction=Restart the server with the clean option.
+
+# The HealthCheck bean call method had an exception
+healthcheck.bean.call.exception.CWMMH0050E=CWMMH0050E: The {0} HealthCheck method in the {1} application from the {2} module threw exception {3} with the following message: {4}
+healthcheck.bean.call.exception.CWMMH0050E.explanation=The specified call method has produced an exception.
+healthcheck.bean.call.exception.CWMMH0050E.useraction=Review your call method within the HealthCheck bean to identify the exception that occurred.
+
+# The HealthCheck bean call method had a DOWN outcome
+healthcheck.application.down.CWMMH0052W=CWMMH0052W: The {0} implementing HealthCheckResponse in the {1} application in module {2}, reported a {3} status with data {4}.
+healthcheck.application.down.CWMMH0052W.explanation=The DOWN status might cause a monitoring service to take action on this server.
+healthcheck.application.down.CWMMH0052W.useraction=No user action is required.  This warning is a record of the DOWN status.
+
+# The Readiness HealthCheck had a DOWN outcome, when applications are not started yet
+readiness.healthcheck.applications.not.started.down.CWMMH0053W=CWMMH0053W: The readiness health check reported a DOWN overall status because the following applications have not started yet: {0}
+readiness.healthcheck.applications.not.started.down.CWMMH0053W.explanation=The specified applications have not completely started yet and are not ready to process requests, hence the readiness health check reported a DOWN status.
+readiness.healthcheck.applications.not.started.down.CWMMH0053W.useraction=No user action is needed, wait until the applications start. This warning is a record of the DOWN status.
+
+# The Startup HealthCheck had a DOWN outcome, when applications are not started yet
+startup.healthcheck.applications.not.started.down.CWMMH0054W=CWMMH0054W: The startup health check reported a DOWN overall status because the following applications have not started yet: {0}
+startup.healthcheck.applications.not.started.down.CWMMH0054W.explanation=The specified applications have not completely started yet and are not ready to process requests, hence the startup health check reported a DOWN status.
+startup.healthcheck.applications.not.started.down.CWMMH0054W.useraction=No user action is needed, wait until the applications start. This warning is a record of the DOWN status.
```

Comment:
wait until the applications start

and then what? 

ie what happens when they do? Does the check report an UP status?

Reply:
Right, as soon as the applications have started, the user-defined health checks will be processed for Startup. If there aren't any, then the overall status will be UP.

Reply:
does it make sense to say:
This warning is a record of the DOWN status and no user action is needed. Wait until all the applications are started. After all the applications are started, the user-defined health checks are processed.

Reply:
Yeah, that makes sense! I will make the changes, as well as there is similar message for the Readiness as well, i will apply the same changes there as well.